test/accreditation/test_accred.py

```diff
@@ -53,56 +50,53 @@ def test_accred_circuits(self):
         target_circuit.h(2)
         target_circuit.h(3)
         target_circuit.measure(q_reg, c_reg)
-
         # make trap circuits
-        circ_list, postp_list, \
-            v_zero = accred.accreditation_circuits(target_circuit, v)
-
-        self.assertEqual(len(circ_list), v + 1,
+        accredsys = accred.AccreditationCircuits(target_circuit)
```

Comment:
The tests should then be updated to use a fixed seed (but different to the seed used in the generation) for accred and simulation as well